apps/api/src/app/subscribers/usecases/delete-subscriber-credentials/delete-subscriber-credentials.usecase.ts

```diff
@@ -0,0 +1,65 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { SubscriberRepository, IntegrationRepository, SubscriberEntity } from '@novu/dal';
+import { AnalyticsService, buildSubscriberKey, InvalidateCacheService } from '@novu/application-generic';
+
+import { DeleteSubscriberCredentialsCommand } from './delete-subscriber-credentials.command';
+import { GetSubscriberCommand, GetSubscriber } from '../get-subscriber';
+
+@Injectable()
+export class DeleteSubscriberCredentials {
+  constructor(
+    private invalidateCache: InvalidateCacheService,
+    private subscriberRepository: SubscriberRepository,
+    private integrationRepository: IntegrationRepository,
+    private analyticsService: AnalyticsService,
+    private getSubscriberUseCase: GetSubscriber
+  ) {}
+
+  async execute(command: DeleteSubscriberCredentialsCommand): Promise<void> {
+    const foundSubscriber = await this.getSubscriberUseCase.execute(
+      GetSubscriberCommand.create({
+        ...command,
+      })
+    );
+
+    if (!foundSubscriber) {
+      throw new NotFoundException(`SubscriberId: ${command.subscriberId} not found`);
+    }
```

Comment:
This check is not needed as `GetSubscriberUseCase` already throws a `NotFoundException` when this case happens.
https://github.com/novuhq/novu/blob/4b71ba025c44a422f6640da3a371a5fdb6470e2d/apps/api/src/app/subscribers/usecases/get-subscriber/get-subscriber.usecase.ts#L14